### PR TITLE
Terse feature list

### PR DESCRIPTION
**Pull request by szepeviktor**
in README

### CODE REVIEW

**Comment by szepeviktor** [comment]
@markoheijnen Could you merge this one?

**Comment by markoheijnen** [comment]
"Remove default filter (hooked in `include/helpers.php`)" should now be remove now right?

**Comment by szepeviktor** [comment]
@markoheijnen I think it is still in the code: https://github.com/markoheijnen/Minit-Pro/blob/master/minit-pro.php#L72-L78

**Comment by markoheijnen** [comment]
I was more thinking to the demo filter. Removing those filters is like removing a feature ^^

**Comment by szepeviktor** [comment]
Should I rephrase it to "Remove filters hooked in include/helpers.php" ?

**Comment by markoheijnen** [comment]
Or something like: Don't add additional comments added in include/helpers.php.

**Comment by szepeviktor** [comment]
Done.

**Comment by szepeviktor** [comment]
Thanks.

**Comment by szepeviktor** [comment]
You may also want to add a repo description here on GitHub. (top of the page)
